conference/NW/mobicom.yml

```diff
@@ -0,0 +1,17 @@
+- title: MobiCom
+  description: ACM Annual International Conference On Mobile Computing And Networking
+  sub: NW
+  rank: A
+  dblp: mobicom
+  confs:
+    - year: 2022
+      id: mobicom22
+      link: https://www.sigmobile.org/mobicom/2022/
+      timeline:
+        - abstract_deadline: '2021-08-13 23:59:59'
+          deadline: '2021-08-20 23:59:59'
+        - abstract_deadline: '2021-03-18 23:59:59'
```

Comment:
I think the year should be 2022? Please check this `deadline` and `abstract_deadline` field ~

Reply:
Absolutely.
I will push a new version.

Reply:
That's nice! We appreciate your contribution.